src/authentic/action_helpers.cr

```diff
@@ -28,7 +28,7 @@ module Authentic::ActionHelpers(T)
   #
   # This method should *not* be overridden. If you want to require a current user,
   # override the `current_user` method (note no `?`).
-  def current_user? : T?
+  memoize def current_user? : T?
```

Comment:
The question mark means that it's not memoizable it seems?

Reply:
Yeah, we can't memoize predicate methods. That would have to be fixed first before we can do that here... I guess I could implement some cheap memoizing on this method with an instance variable?